programs/clearing_house/src/controller/orders.rs

```diff
@@ -1612,6 +1614,22 @@ pub fn fulfill_order_with_match(
             update_quote_asset_amount(referrer_position, market, cast(referrer_reward)?)?;
 
             referrer_stats.increment_total_referrer_reward(cast(referrer_reward)?)?;
+
+            if now > referrer_stats.next_epoch_ts {
```

Comment:
should maybe put this on the referrer_stats struct. can either be a separate method or we can just have one `update_referrer_reward` function which takes in a timestamp and referrer reward?